### PR TITLE
[#1017] Menstrual Survey trigger changes

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/1017

ChangeLog

- Change the default value of the 459098666 (Menstrual Survey Status) from 972455046 (Not Started) to 789467219 (Not Yet Eligible)
- Add code to our notification logic that will change 459098666 (Menstrual Survey Status) to 972455046 (Not Started) upon first email notification

### CODE REVIEW

File: utils/notifications.js [hunk 1]
```diff
@@ -489,6 +489,30 @@ async function getParticipantsAndSendNotifications({ notificationSpec, cutoffTim
       }
     }
```
[the remark this answers is not in the document]
Comment (jhflorey) [reply utils/notifications.js:491]: I'm not either but as i know   the notificationSpecs collection in  firestore we defined 2 languages for email.

**Comment by we-ai** [comment]
This PR may not be needed:
-  For CID 459098666 (Menstrual Survey Status), 972455046 (Not Started) is more reasonable used as the default value  than 789467219 (Not Yet Eligible). CID 289750687 (Eligible for menstrual cycle survey) stores eligibility already.
- There's no need to set value of CID 459098666 (Menstrual Survey Status) from 789467219 (Not Yet Eligible) back to 972455046 (Not Started), if 972455046 (Not Started) is kept as the default value.
- In PWA, delayed Menstrual Survey can be achieved by checking the submission date of biospecimen survey when loading pages.

**Comment by anthonypetersen** [comment]
@we-ai that is fine to do, but the requirement that needs to be met is the survey becoming available at the time the first reminder email is sent, not just anytime on the day that is has been five days since survey completion

**Comment by we-ai** [comment]
Thanks for the info @anthonypetersen. Without mass updating participant data, there might be 2 ways to do it:
- It may be better to open the survey at beginning of day 5. We can have instruction (when biospecimen survey is submitted or at other occassions) that this survey is available at day 5. If the survey is finished timely, the reminder won't be sent at 3pm on day 5.
- If we want to match the time of survey reminder and survey, assuming notifications are sent out without issues, we can open the survey at 3pm on day 5 by checking participant data.

**Pull request closed without merging**